CONTRIBUTING.md

```diff
@@ -147,6 +147,29 @@ The current code conventions for the source files are as follows:
      * One-line non-documentation comments, intended for schema developers only,
        must use the `// ...` style.
 
+<a name="documentation"></a>
+## Documentation
+
+The goal of GA4GH is to define an interoperable API specification.  To achieve
+this intent, rational, and semantics of all data objects and operations
+need to be clearly and precisely defined. Decisions that are not captured in
+documentation are lost.
+
+All schemas defined in GA4GH must include normative documentation.  This
+should consisted of overview and design documentation as well as documentation
+in Avro schemas.  This documentation should explain the gaols, overall design
+and rational for decisions that were made.  It must be addressed to both
+client and server developer audience.  It may cite publish papers or stable
+web documentation.
+
+Documentation must integrate with the Sphinx-base documentation system
```

Comment:
This process and directory do not actually exist in master yet. Perhaps remove this paragraph and merge as part of the documentation branch PR?